package.json

```diff
@@ -123,8 +123,7 @@
   "pre-commit": {
     "colors": true,
     "run": [
-      "lint:pre-commit",
-      "storybook:visual-docker"
```

Comment:
This shouldn't be running storybook on a pre-commit hook

Reply:
This should be moved to separate PR

Reply:
It already was - otherwise you wouldn't be able to commit anything 😉 
https://github.com/SAP/fundamental-styles/pull/1067